cmd/check.go

```diff
@@ -81,6 +81,11 @@ You may specify one or more benchmarks or controls to run (separated by a space)
 		AddStringArrayFlag(constants.ArgSnapshotTag, "", nil, "Specify tags to set on the snapshot").
 		AddStringFlag(constants.ArgSnapshotLocation, "", "", "The cloud workspace... ")
 
+	cmd.AddCommand(listSubCmd(listSubCmdOptions{
+		shortDescription: "",
```

Comment:
can we put in placeholder descriptions. Can we use the same description for all parent commands (i.e. can we avoid having to pass it in?)

Reply:
so can we remove `shortDescription: "",` etc? from the code?